k8s/stubbornjava.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stubbornjava-deployment
+  labels:
+    app: sj-web
+spec:
+  replicas: 1
```

Comment:
Recommend defining your update strategy once you're considering having the replicas go above 1.